scripts/devel/create_devel_task_generator.py

```diff
@@ -186,6 +191,7 @@ def main(argv=sys.argv[1:]):
         'install_lists': [],
 
         'testing': args.testing,
+        'abichecking': args.abichecking,
```

Comment:
Nitpick: please match the key with the argument `run_abichecker`.

Reply:
7d60961